tests/test_headers/string.h

```diff
@@ -0,0 +1,19 @@
+#include "common.h"
```

Comment:
Not 100% sure what your reason for having this in a header way but, due to the horror/magic within gtest, it can find tests across multiple executable units so there's no harm having tests in a bunch of cc files (this is how GeNN's unit tests are structured)

Reply:
My reason for having it in a header is not really understanding gtest properly :wink:. It's cool that you can have multiple cc files though... I'll tidy this up then.